test/lib/BedrockTester.cpp

```diff
@@ -501,16 +501,20 @@ int BedrockTester::getControlPort() {
     return _controlPort;
 }
 
-bool BedrockTester::waitForState(string state, uint64_t timeoutUS) {
-    return waitForStates({state}, timeoutUS);
+int BedrockTester::getCommitCount() {
```

Comment:
I don't think this function is required.

Reply:
Totally, it wasn't reliable, so I stopped using it. Removing.